manifests/05-operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: pagerduty-operator
       containers:
         - name: pagerduty-operator
-          image: quay.io/app-sre/pagerduty-operator
+          image: quay.io/lpieri/pagerduty-operator-metrics:metrics
```

Comment:
I think you left in some testing stuff here. Please revert changes to this file

Reply:
Yes this is temporary for testing purposes.